HelpSource/Classes/Node.schelp

```diff
@@ -181,11 +181,11 @@ discussion::
 This is similar to map above, but you specify the number of sequential Controls to map. If this Node is a Group this will mapn all Nodes within the Group.
 
 method:: release, releaseMsg
-This is a convenience method which assumes that the synth contains an envelope generator (an EnvGen, Linen, or similar UGen) running a sustaining envelope (see Env) and that it's gate argument is set to a control called \gate. This method will cause the envelope to release.
+This method causes the receiver to be freed over the specified amount of time. This is a convenience method which assumes that the synth contains an envelope generator (an EnvGen, Linen, or similar UGen) running a sustaining envelope (see Env) and that it's gate argument is set to a control called \gate.
```

Comment:
I wouldn't understand what "to be freed over the specified amount of time" should mean. Freeing is usually a moment, and also technically it is.

Reply:
a suggestion: "This method causes the receiver to be freed after the specified amount of time. It sets the code::\gate:: argument to a value that will cause envelopes (suc as EnvGen or Linen) to release over the specified time."